docs/index.rst

```diff
@@ -220,114 +212,44 @@ $ pytest tests/foreman/cli/test_host.py -n 4
 
 For more information about Python's `pytest`_ module, read the documentation.
 
-Testing With Unittest
```

Comment:
Most of the changes in this file were to resolve docs build failures, but Unittest/Nosetest could have been removed with the final pytest PR, so removing them here.